mongodbatlas/fw_data_source_mongodbatlas_alert_configurations.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
 	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
 	"github.com/hashicorp/terraform-plugin-framework/types"
-	matlas "go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/atlas-sdk/v20230201006/admin"
```

Comment:
```suggestion
	"go.mongodb.org/atlas-sdk/v20230201008/admin"
```

Can we use the latest version? https://github.com/mongodb/atlas-sdk-go/releases/tag/v20230201008.0.0

This can be addressed in a followup PR